### PR TITLE
swaps: more input flip & max

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes TEAM2-217

## What changed (plus any additional context for devs)

handle all cases for `flip` and `max` in mainnet and L2s including arbitrum

## PoW (screenshots / screen recordings)

https://user-images.githubusercontent.com/12115171/176963577-98cf3d87-2eea-4e8a-bd63-e0cebc1d0ecf.mp4

## Dev checklist for QA: what to test

- flip
- max
- type fast and delete characters to check if the input is not flickering
- 
## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Review by derHowie — APPROVED** [review]
Test cases worked for me. 🥇

File: src/components/exchange/ExchangeNativeField.js [hunk 1]
```diff
@@ -76,6 +78,12 @@ const ExchangeNativeField = (
     return colors.alpha(color, opacity);
   }, [colors, isFocused, nativeAmount]);
 
+  useEffect(() => {
+    setEditing(true);
+    startTimeout(() => setEditing(false), 1000);
```
Comment (benisgold) [inline src/components/exchange/ExchangeNativeField.js:83]: 🔥

**Comment by benisgold** [comment]
The only (small) issue i can see is that if you click max and then flip, the flipped number isn't exactly the same. 

https://user-images.githubusercontent.com/15272675/177013698-841791f4-9bca-4047-9760-01fbd1521569.mp4

**Comment by estebanmino** [comment]
nice catch @benisgold, fixed in the last commit

this will round every flip value to 6 decimals
https://user-images.githubusercontent.com/12115171/177367871-f72f6dcc-dc95-4cf6-a31b-020c3ee1bca1.mp4

**Comment by benisgold** [comment]
your fix on the previous issue looks good. 1 more thing: seems like flipping stops working once you receive the "Insufficient ETH" warning when trying to max out ETH.

https://user-images.githubusercontent.com/15272675/177378734-a44648cd-4da6-4f23-934c-51c733bc6f75.mp4

**Pull request merged**